### ISSUE TITLE
decoding errors not explained in wiki

### ISSUE THREAD

**Issue by LeonRonin** [szechyjs/dsd#85]
hi
what are these errors that i'm getting while decoding DMR? you haven't explained them in the wiki:
+DMR    MS VC2    e:5m7r7r
+DMR    MS VC3    e:7r7m7r
+DMR    MS VC4    e:7r7r7m
+DMR    MS VC5    e:7r6r7r
+DMR    MS VC6    e:6m7r7r
+DMR    MS VOICE  e:6r5m6r
+DMR    MS VC2    e:7r7r7m
+DMR    MS VC3    e:7r7r7r
+DMR    MS VC4    e:6m7r6r
+DMR    MS VC5    e:7r7m7r
+DMR    MS VC6    e:7r7r5m
+DMR    MS VOICE  e:7r7r7r
+DMR    MS VC2    e:7m7r5r
+DMR    MS VC3    e:5r6m7r
+DMR    MS VC4    e:6r7r7m
I don't get any voice out dsd by the way...
thank you

**Comment by szechyjs** [comment]
What version of DSD are you using? This output doesn't seem to match this version, are you using the DSDPlus fork?

**Comment by LeonRonin** [comment]
yes i'm using the dsdplus fork. on some other DMR reception I don't get any decoding by dsdplus. I know the signal is DMR. could it be the signal being too strong because the radio is too close?
